### DISCUSSION TITLE
VIRTUAL ASSET DERIVATIVES

### DISCUSSION THREAD

**Discussion by Aaron-XReg** [MinistryFinancialServices/Virtual-Asset-Amendment-Consultation#43] [General]
Virtual asset (VA) derivatives play an increasingly central role in the virtual asset market and are a staple on most major VA trading platforms and in all VA platforms’ future plans. Progressive jurisdictions, including Bermuda, have recognised this and explicitly included VA derivatives within the scope of their virtual asset legislation, attracting important market participants to their jurisdiction. 

For VA trading platforms planning to offer or already offering VA derivatives in Cayman under the VASP Act, there is currently a level of uncertainty with respect to whether authorisation or exemption under the Securities Investment Business Act (SIBA) will be required. The VASP Act and proposed CIMA rules and guidance for VA trading platforms represent one of the most well-designed and proportionate regimes for VA trading platforms in the world. Conversely, SIBA does not include a trading platform regime. Therefore, there would be no regulatory benefit to requiring dual registration, and the mechanisms of the Act which require CIMA to undertake this analysis for trading platforms are unnecessary. 

The VASP Act should be amended to bring trading platforms that offer VA derivatives exclusively within its scope. This would be a relatively minor fix that would significantly enhance the clarity and transparency of the regulatory treatment of a core product line for VA trading platforms. This added clarity will encourage leading global VA trading platforms to choose the Cayman Islands for their operations and help to develop the jurisdiction as a global hub for this critical market infrastructure.

**Comment by hubXglobal** [comment]
Without regulatory clarity on crypto derivatives global exchanges would not be interested in entering the Cayman market.

**Reply by AGIAI** [reply]
Agreed

**Comment by Clearwaterscrypto** [comment]
Being able to offer both spot trading and VA derivatives under one licence is incredibly valuable for service providers and is often a key deciding factor for exchanges looking for a good jurisdiction from where to operate. This would not only align Cayman to Bermuda, but also The Bahamas where derivatives are clearly being brought within of their VA regime via the Digital Assets and Registered Exchanges Bill 2023.
